src/coreclr/tools/Common/System/Collections/Generic/ArrayBuilder.cs

```diff
@@ -46,7 +46,7 @@ public void Add(T item)
         }
 
 #if NET
-        public readonly Span<T> AsSpan(int start) => _items.AsSpan(start);
+        public readonly Span<T> AsSpan(int start) => _items.AsSpan(start, _count);
```

Comment:
I believe this should be:

```suggestion
        public readonly Span<T> AsSpan(int start) => _items.AsSpan(start, _count - start);
```

Reply:
Yes I've also caught this.